pull/context.go

```diff
@@ -65,6 +65,9 @@ type Context interface {
 	// CreatedAt returns the time when the pull request was created.
 	CreatedAt() time.Time
 
+	// State returns the state of the pull request (open or closed)
+	State() string
```

Comment:
Since this is either `open` or `closed` is there an advantage to returning it as a string instead of implementing it as an `IsClosed()` or `IsOpen()` method?

Reply:
Yup, that seems like a better option. I thought I saw that `merged` was an option somewhere, but cannot find the docs I had seen it in. Right now [this](https://docs.github.com/en/rest/reference/pulls#update-a-pull-request) is the best enumeration of state values I've been able to find.